src/queries.ts

```diff
@@ -58,155 +57,31 @@ export async function getAllArDriveTransactionsWithin(range: HeightRange): Promi
 	console.log('Height ranges to query are', nonCachedRanges.length);
 
 	for (const nonCachedRange of nonCachedRanges) {
-		let hasNextPage = true;
-		let cursor = '';
+		let height = nonCachedRange.min;
 
 		console.log('Querying range', nonCachedRange);
 
-		while (hasNextPage) {
-			const query = createQuery(nonCachedRange, cursor);
-			const response = await sendQuery(query);
-			const edges = response.edges;
-			hasNextPage = response.pageInfo.hasNextPage;
-			console.log(` # Recieved ${edges.length} transactions.`, { hasNextPage });
+		while (height <= nonCachedRange.max) {
+			const edges = await getAllParsedTransactionsOfBlock(height);
+			const ardriveEdges = edges.filter(ardriveTxFilter);
+			console.log(` # Recieved ${ardriveEdges.length} transactions.`, { height });
+
+			height++;
 
-			if (!edges.length) {
-				cache.setEmptyRange(nonCachedRange);
+			if (!ardriveEdges.length) {
+				cache.setEmptyRange(new HeightRange(height, height));
 				continue;
 			}
 
 			// to ensure we are querying in descendant order
-			const oldestTransaction = edges[edges.length - 1];
-			const mostRecentTransaction = edges[0];
+			const oldestTransaction = ardriveEdges[ardriveEdges.length - 1];
+			const mostRecentTransaction = ardriveEdges[0];
 			new HeightRange(oldestTransaction.node.block.height, mostRecentTransaction.node.block.height);
 
-			cursor = oldestTransaction.cursor;
-			await cache.addEdges(edges);
+			await cache.addEdges(ardriveEdges);
 		}
 	}
 	cache.done();
 	const allEdges = await cache.getAllEdgesWithinRange();
 	return allEdges;
 }
-
-/**
- * Runs the given GQL query
- * @param query the query object
- * @throws if the GW returns a syntax error
- * @returns {GQLTransactionsResultInterface} the returned transactions
- */
-async function sendQuery(query: Query): Promise<GQLTransactionsResultInterface> {
```

Comment:
I don't think it was strictly necessary to remove all this. If GQL becomes viable again, we'd still want to use this.